submissions/aymkin/HTML-CSS-Popup/index.html

```diff
@@ -93,13 +93,14 @@
             </li>
             <li class="nav__element tiel">
                 <input
-                    type="checkbox"
-                    class="nav__element pop-check"
-                    name="tiel"
+                type="checkbox"
+                class="nav__element pop-check"
+                name="tiel"
                 />
+                <div class="outline"></div>
+                <label for="tiel" class="pop-label"></label>
                 <div class="white-angle"></div>
```

Comment:
Triangles better create by pceudoelements `::after` or `::before`.

Reply:
is that mandatory or optionally to create triangles by `::before` and `::after ?`